content/participants/ttjc@Mojojojooo/index.md

```diff
@@ -35,6 +34,28 @@ It would be helpful if I understood front end and backend development.
 
 ### projects
 
-not a lot of projects are done. I will keep updating it with real links when done.
+•Waste management using Arduino.
```

Comment:
Hey! @Mojojojooo Please follow the proper format from the registration video to add projects in this file. Each project should have the following:
```
#### Project Title here
_description_
_stack_
_hosted link_
_github link_
```
Also keep in mind that *only* projects done during the job challenge are accepted for the challenge. So only include those projects here.